tests/isVisible.spec.ts

```diff
@@ -50,7 +50,10 @@ describe('isVisible', () => {
         }
       }
     }
-    const wrapper = mount(Comp)
+    // TODO: attachTo can be removed when https://github.com/jsdom/jsdom/issues/3502 is fixed
+    // with jsdom v21.1, getComputedStyle is cached and not updated for elements not in the DOM
+    // so the test fails (because the element is checked a first time)
+    const wrapper = mount(Comp, { attachTo: document.body })
```

Comment:
Nice. I almost wonder if attachTo: document.body should be the default behavior. I wonder if that'd be a breaking change?

Reply:
Maybe we should, I don't know what are the downsides (most testing libs in other ecosystems do attach components right?). As you have more context, do you know why this is not the case in VTU?